### PR TITLE
Added custom Banner

### LINKED ISSUE #252: Using Spring Boot Custom Banners
https://www.baeldung.com/spring-boot-disable-banner

https://www.baeldung.com/spring-boot-custom-banners

### PR DESCRIPTION
**Pull request by ParthAdhana**
This PR addresses "Using Spring Boot Custom Banners #252" issue . 
The custom Banner is present in Banner.txt file along with application.yaml file

Closes #252

### CODE REVIEW

**Comment by FanJups** [comment]
Hi @ParthAdhana , I am so happy for your contribution. We will review it together and merge asap 😊💯

**Comment by FanJups** [comment]
Closes #252

**Comment by ParthAdhana** [comment]
Hi @FanJups , Thanks for the kind word , please let me know if any corrections are to be made in this PR :)

File: .history/jsgenerator-slim-api/src/main/resources/banner_20230814060638.txt
Comment (FanJups) [inline .history/jsgenerator-slim-api/src/main/resources/banner_20230814060638.txt:1]: The goal here is to use the name of the module, in this case jsgenerator-api, jsgenerator-cli, ...

Comment (FanJups) [reply .history/jsgenerator-slim-api/src/main/resources/banner_20230814060638.txt:1]: Turning these names into ascii art with this website  https://patorjk.com/software/taag/#p=display&f=Big&t=JSGENERATOR-API

https://patorjk.com/software/taag/#p=display&f=Big&t=JSGENERATOR-CLI

**Comment by FanJups** [comment]
> Hi @FanJups , Thanks for the kind word , please let me know if any corrections are to be made in this PR :)

https://www.baeldung.com/spring-boot-custom-banners

First, just add this property into application.yaml  of slim api and slim cli

```yaml
spring:
  banner:
    location: classpath:/banner.txt
```

Then, create src/main/resources/banner.txt  in these 2 modules.

Go to https://patorjk.com/software/taag/#p=display&f=Big&t=JSGENERATOR-API to get the content for api  and  https://patorjk.com/software/taag/#p=display&f=Big&t=JSGENERATOR-CLI for cli modules.

Copy and paste the result into banner.txt of each module.

**How to know if everything work well ?**

Just run these commands for api `--projects jsgenerator-api spring-boot:run` and cli `--projects jsgenerator-api spring-boot:run` at the root directory js-generator. Then, just pay attention to console output.

**Comment by FanJups** [comment]
@ParthAdhana any issues ? I am here to help.

**Comment by ParthAdhana** [comment]
@FanJups I have created another PR and this time from a feature branch can you please check that one out,  #256 I am not sure if that is the correct way of doing that , as it is still my first PR.

**Pull request closed without merging**